src/cloud-api-adaptor/versions.yaml

```diff
@@ -34,7 +34,7 @@ git:
     reference: main
   guest-components:
     url: https://github.com/confidential-containers/guest-components
-    reference: v0.10.0
+    reference: c2022037d8fbb076f569529e93b2cbe63a3968bb
```

Comment:
Hi @stevenhorsman !

Tests passing, great!

IIUC from the meeting today, using that commit it will pull in some changes that break the kbs <-> guest-components integration (if kbs is older). So I suggested to branch off v0.10.0, cherry-pick only that fix and release v0.10.1. What do you have in mind? Point to that commit anyway and just document the version of kbs that should be deployed?

Reply:
I don't think there is a protocol breaking change since 0.10.0 and the fact that the tests pass backs this up, so I don't think it's worth adding complexity to things for this.

Reply:
Roger that!